Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -297,16 +330,24 @@ void Jit64::psq_lXX(UGeckoInstruction inst)
 	// In memcheck mode, don't update the address until the exception check
 	if (update && !jo.memcheck)
 		MOV(32, gpr.R(a), R(RSCRATCH_EXTRA));
-	MOV(32, R(RSCRATCH2), Imm32(0x3F07));
 
-	// Get the high part of the GQR register
-	OpArg gqr = PPCSTATE(spr[SPR_GQR0 + i]);
-	gqr.AddMemOffset(2);
+	if (gqrIsConstant)
+	{
+		GenQuantizedLoad(w, (EQuantizeType)(gqrConstant & 0x7), (gqrConstant & 0x3F00) >> 8);
```

Comment:
w is typeof int while the function takes a bool as parameter.
Windows builds are failing because of that.